pages/index.js

```diff
@@ -1,44 +1,83 @@
 import ListEntry from '../components/ListEntry';
 import {
   StyledButton,
-  StyledH1Home,
   StyledUl,
   StyledTextHome,
+  StyledSearch,
+  StyledPageContainerHome,
 } from '../components/StyledComponents';
 
+import { useState, useEffect } from 'react';
 import Link from 'next/link';
+import useHydration from '../hooks/useHydration';
 import useStore from '../hooks/useStore';
 import PlusIcon from '../public/plusicon.svg';
+import HeaderHome from '../components/HeaderHome';
```

Comment:
I see 'HeaderHome in components', I see 'StyledHeaderHome in StyledComponents'. Confusing. I also see a huge amount of imports and exports of styled components. To me it seems like the way you are handling them could be simplified. Just a thought...